attunity_exporter.go

```diff
@@ -11,15 +11,16 @@ import (
 	"github.com/sirupsen/logrus"
 	"gopkg.in/yaml.v2"
 
-	"github.com/wbh1/attunity_exporter/collector"
 	"github.com/prometheus/client_golang/prometheus"
+	"github.com/wbh1/attunity_exporter/collector"
 )
 
 var (
-	cfg     = collector.Config{}
-	port    = kingpin.Flag("port", "the port to bind the exporter to.").Default("9103").String()
-	cfgFile = kingpin.Flag("config-file", "path the config.yml file").Default("config.yml").String()
-	debug   = kingpin.Flag("debug", "Enable debug logging").Bool()
+	cfg        = collector.Config{}
+	port       = kingpin.Flag("port", "the port to bind the exporter to.").Default("9103").String()
+	cfgFile    = kingpin.Flag("config-file", "path the config.yml file").Default("config.yml").String()
+	debug      = kingpin.Flag("debug", "Enable debug logging").Bool()
+	ignoreCert = kingpin.Flag("ignore-cert", "Will ignore certificate warnings").Bool()
```

Comment:
I'd prefer this was a directive in the config file, but also I don't think I'll ever have to use this project again, so have at it 😁

Reply:
This flag can be dropped now.